components/scripts/lib/info.sh

```diff
@@ -172,6 +174,35 @@ print_realized_build_time_savings() {
   fi
 }
 
+print_potential_build_time_savings() {
```

Comment:
0: Technically, this shows the potential build time savings that could be realized only from build caching and parallel task execution.  It doesn't take into account other potential build time savings, such as those gained using Test Distribution or Predictive Test Selection.

I wonder if we should clarify this somehow. Or make it clearer that, even if the "potential build time savings" have been fully realized that there are still ways in which to further optimize and improve the build.

Reply:
I will defer this to @etiennestuder.

Reply:
We had a longer discussion about this and came up with a proposal that will be implemented in a separate PR.